### ISSUE TITLE
Useless padding on Ubuntu

### ISSUE THREAD

**Issue by PeyTy** [danielchatfield/trello-desktop#12]
I see, this is for frameless OSX windows, but makes no effect on Ubuntu.

![image](https://cloud.githubusercontent.com/assets/3642643/19492375/154395e2-957f-11e6-8196-1cc4c9830ac1.png)

**Comment by yurivecchi** [comment]
We can remove the titlebar and make our own buttons in this area, this way the app will be consistent in all plataforms

**Comment by brainscript** [comment]
@yurivecchi Is this already in release 0.1.1? If yes, it's not fixed.
![bildschirmfoto vom 2016-12-26 12-44-30](https://cloud.githubusercontent.com/assets/8706734/21481541/15020fc6-cb69-11e6-8bec-2e4bf9d50093.png)

**Comment by yurivecchi** [comment]
@brainscript no, it isn't, I'm testing here and sometimes the custom buttons I made disappear leaving the blank space, and there are others problems with the components, I will work on it this week probably.

**Comment by brainscript** [comment]
@yurivecchi Ok, thanks! :smile_cat:
